Airports/EGPK/Positions.txt

```diff
@@ -1,4 +1,4 @@
 EGPK_APP:Prestwick Radar:129.450:PKR:P1:EGPK:APP:-:-:4501:4520:N055.30.28.000:W004.35.08.000::
-EGPK_F_APP:Prestwick Director:124.625:PKF:P5:EGPK:APP:-:-:4501:4520:N055.30.28.000:W004.35.08.000::
+EGPK_F_APP:Prestwick Director:124.630:PKF:P5:EGPK:APP:-:-:4501:4520:N055.30.28.000:W004.35.08.000::
 EGPK_TWR:Prestwick Tower:118.150:PKT:P2:EGPK:TWR:-:-:4501:4520:N055.30.28.000:W004.35.08.000::
 EGPK_ATIS:Prestwick ATIS:121.125:PKI:S:EGPK:ATIS:-:-::::::
```

Comment:
```suggestion
EGPK_ATIS:Prestwick ATIS:121.130:PKI:S:EGPK:ATIS:-:-::::::
```
According to [the AIP](https://www.aurora.nats.co.uk/htmlAIP/Publications/2023-11-02-AIRAC/html/eAIP/EG-AD-2.EGPK-en-GB.html#EGPK-AD-2.18)

Reply:
Updated